applications/callflow/src/module/cf_directory.erl

```diff
@@ -139,16 +139,22 @@ handle(Data, Call) ->
 -spec directory_start(whapps_call:call(), directory(), directory_users()) -> 'ok'.
 directory_start(Call, State, CurrUsers) ->
     _ = whapps_call_command:flush_dtmf(Call),
-    {'ok', DTMF} = play_directory_instructions(Call, sort_by(State)),
-    case whapps_call_command:collect_digits(100, ?TIMEOUT_DTMF, ?TIMEOUT_DTMF, Call) of
-        {'error', _E} ->
-            lager:error("failed to collect digits: ~p", [_E]),
-            cf_exe:stop(Call);
-        {'ok', <<>>} ->
-            whapps_call_command:audio_macro([{'prompt', ?PROMPT_SPECIFY_MINIMUM}], Call),
-            directory_start(Call, State, CurrUsers);
-        {'ok', DTMFS} ->
-            maybe_match(Call, add_dtmf(add_dtmf(State, DTMF), DTMFS), CurrUsers)
+    case play_directory_instructions(Call, sort_by(State)) of
+	{'error', _Error} ->
+	    lager:error("failed to collect digits: ~p", [_Error]),
+	    cf_exe:stop(Call);
+
+	{'ok', DTMF} ->
+	    case whapps_call_command:collect_digits(100, ?TIMEOUT_DTMF, ?TIMEOUT_DTMF, Call) of
```

Comment:
I'd like to see this in a sub function, maybe something like `collect_digits(Call, State, CurrUsers, DTMF)`.